<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="BG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/image/src/Back_final.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lable_select">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/src/select.png</pixmap>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/number 3.png</normaloff>:/src/number 3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/number4.png</normaloff>:/src/number4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/number 5.png</normaloff>:/src/number 5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/number 6.png</normaloff>:/src/number 6.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="cursorWidth">
     <number>3</number>
    </property>
    <property name="placeholderText">
     <string>Player 1 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 2 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 3 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 4 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 5 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 6 Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 1 Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 2 Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 3 Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 4 Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 5 Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Age_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/src/LineEdit_selected2.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="placeholderText">
     <string>Player 6 Age</string>
    </property>
   </widget>
   <widget class="QComboBox" name="color_1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="color_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="go_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/Gobtn.png</normaloff>:/src/Gobtn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
